system-int-test/resources/shapefiles/antimeridian.geojson

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Format this file to make it easier to read (pipe it through jq). Same comment for all the other shapefile examples that aren't formatted nicely.

Reply:
Done